### DISCUSSION TITLE
Installing on Monterey

### DISCUSSION THREAD

**Discussion by galleon** [stella-cv/stella_vslam#254] [Q&A] [answered]
Hi

I am new to OpenVSLAM. Has anyone successfully installed it on MacOS Monterey ? When trying to follow the instructions for g2o I got the following error despite libomp being installed with brew  (and omp.h is indeed in  /usr/local/Cellar/libomp/13.0.1/include).

[Edit] removing `libomp` and compiling main branch works fine !

[ 14%] Building CXX object g2o/core/CMakeFiles/core.dir/cache.cpp.o
In file included from /Users/alleon_g/code/g2o/g2o/core/cache.cpp:27:
In file included from /Users/alleon_g/code/g2o/g2o/core/cache.h:32:
In file included from /Users/alleon_g/code/g2o/g2o/core/optimizable_graph.h:34:
/Users/alleon_g/code/g2o/g2o/core/openmp_mutex.h:33:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
1 error generated.
make[2]: *** [g2o/core/CMakeFiles/core.dir/cache.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /Users/alleon_g/code/g2o/g2o/core/hyper_graph_action.cpp:28:
In file included from /Users/alleon_g/code/g2o/g2o/core/optimizable_graph.h:34:
/Users/alleon_g/code/g2o/g2o/core/openmp_mutex.h:33:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
1 error generated.

**Comment by ymd-stella** [comment, marked as answer]
> [Edit] removing libomp and compiling main branch works fine !
